webpack/mocha-require.js

```diff
@@ -24,8 +25,11 @@ global.matchMedia = () => ({
 global.jQuery = global.$ = require("jquery");
 require("../kitsune/sumo/static/sumo/js/i18n");
 global.gettext = dom.window.gettext;
-global.jsdom = dom
+global.interpolate = dom.window.interpolate;
+global.jsdom = dom;
 global.window.fetch = () => {};
 global.CustomEvent = dom.window.CustomEvent;
 global.requestAnimationFrame = () => {};
 global.window.waffle = { flag_is_active: () => undefined };
+global.window.URL.createObjectURL = () => {};
+global.window.open = () => {};
```

Comment:
```suggestion
global.window.open = () => {};
global.Blob = dom.window.Blob;
```

Without this line I'm getting a failing test and this error:

```sh
Error: Uncaught [ReferenceError: Blob is not defined]
```

Reply:
Interesting - I wonder why I wasn't hitting that? Weird.